### PR TITLE
Create HOL 0 for setting up VSTS team project

### PR DESCRIPTION
**Pull request by sawpresto**
Add a HOL for setting up the VSTS team project and creating a PAT.

### CODE REVIEW

File: docs/HOL_Set-Up-MRP/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+![](<media/clone_mrp.png>)
+
+**NOTE:** If you are running Linux on your local workstation, install git:
+
+    sudo apt-get install git
+	
+**NOTE:** If you are running Windows, you can install the git client from here:
+
+	http://git-scm.com/download
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Change for a Hyperlink 
```
[http://git-scm.com/download ](http://git-scm.com/download)
```
[http://git-scm.com/download ](http://git-scm.com/download )

File: docs/HOL_Set-Up-MRP/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+**3.** Add the Clone URL of your Visual Studio Team Services repository as a new remote called **vsts** and push to it
+your Visual Studio Team Services account. While pushing, use any username without the "@" symbol and the PAT that you copied as the password.
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Previously copied in your clipboard ( Step 4 )

File: docs/HOL_Set-Up-MRP/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+In this lab, we will set up the PartsUnlimited MRP VSTS team project as a pre-requisite for the Continuous Integration lab. You will need an active Visual Studio Team Services (VSTS) account already, or you will need to sign up for a free Visual Studio Team Services account by following [this link.](https://www.visualstudio.com/en-us/docs/setup-admin/team-services/sign-up-for-visual-studio-team-services) Ensure that you are also a Project Collection Administrator for the Visual Studio Team Services account or follow [this link.](https://www.visualstudio.com/en-us/docs/setup-admin/add-administrator-tfs) 
+
+###Pre-requisites:###
+
+-   An active Visual Studio Team Services (VSTS) account
+
+-   Project Collection Administrator rights to the Visual Studio Team Services account
+
+### Tasks Overview: ###
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Task

File: docs/HOL_Set-Up-MRP/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+![](<media/copy_vsts_repo_url.png>)
+
+**5.** Now that we have the URL for the VSTS repo, we need to authenticate access to  a Visual Studio Team Services personal access token. Follow the steps in this [link](https://www.visualstudio.com/en-us/docs/setup-admin/team-services/use-personal-access-tokens-to-authenticate) 
+ to create a personal access token (PAT).
+
+**NOTE:** These Personal Access Token (PAT) will be used when interacting with the git repository from the
+command line. Make sure you copy the token now. Visual Studio Team Services does not store it and you won't be able to see it again. 
+
+
+**6.** Clone the **PartsUnlimitedMRP** git repository located in GitHub onto your local machine:
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Add something like : Open a command line / terminal on your machine and then ...

File: docs/HOL_Set-Up-MRP/README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+    git clone https://github.com/Microsoft/PartsUnlimitedMRP.git
+
+![](<media/clone_mrp.png>)
+
+**NOTE:** If you are running Linux on your local workstation, install git:
+
+    sudo apt-get install git
+	
+**NOTE:** If you are running Windows, you can install the git client from here:
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: @OguzPastirmaci Do you think we need to explain how install Git on Windows an make sure the git command in on the PATH variable etc.... ?

File: docs/HOL_Set-Up-MRP/README.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+	http://git-scm.com/download
+
+**3.** Add the Clone URL of your Visual Studio Team Services repository as a new remote called **vsts** and push to it
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Maybe add something like : Add the Clone URL of your Visual Studio Team Services repository Previously copied in your clipboard ( Step 4 )

File: docs/HOL_Set-Up-MRP/README.md (hunk 2 above)
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Please be more specific on : While pushing, use any username without the "@" symbol and the PAT that you copied as the password. 
Maybe One screenshot ?

File: docs/HOL_Set-Up-MRP/README.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+	git push -u vsts --all
+	
+![](<media/push_to_vsts.png>)
+
+**NOTE:** We added the Visual Studio Team Services repository as a remote named **vsts**, so we need to
+push to that remote in the future for our changes to appear in our Visual Studio Team Services
+repository.
+
+**4.** Your Visual Studio Team Services account should now have a copy of the PartsUnlimitedMRP
```
Comment (julienstroheker) [inline docs/HOL_Set-Up-MRP/README.md]: Maybe add something like : Refresh the 'Code' section page....

File: docs/HOL_Set-Up-MRP/README.md (hunk 1 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: Or directly to the Windows installation with https://git-scm.com/download/win

File: docs/HOL_Set-Up-MRP/README.md (hunk 5 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: The default options with the installer should set them automatically so we may not need it.

File: docs/HOL_Set-Up-MRP/README.md (hunk 4 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: Good point.

File: docs/HOL_Set-Up-MRP/README.md (hunk 6 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: Good point.

File: docs/HOL_Set-Up-MRP/README.md (hunk 2 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: Good point.

File: docs/HOL_Set-Up-MRP/README.md (hunk 7 above)
Comment (OguzPastirmaci) [inline docs/HOL_Set-Up-MRP/README.md]: Good point.

**Pull request merged**